packages/sync/src/useMultiplayerSync.ts

```diff
@@ -155,13 +167,32 @@ export function useMultiplayerSync(
 	)
 }
 
-/** @public */
+/**
+ * Options for the {@link useMultiplayerSync} hook.
+ * @public
+ */
 export interface UseMultiplayerSyncOptions {
+	/**
+	 * The URI of the multiplayer server. This must include the protocol, e.g. `wss://server.example.com` or `http://localhost:5858`.
```

Comment:
I feel like we should maybe make it clear that this should point not just to the multiplayer server, but to the actual room itself
```suggestion
	 * The URI of the multiplayer server. This must include the protocol, e.g. `wss://server.example.com/my-room` or `http://localhost:5858/my-room`.
```

I also wonder if having the `http` and `wss` examples here is confusing - is there a difference between those? maybe we should call out that although you can pass `http`/`https`, we always rewrite it to `ws`/`wss`